frontend/src/component/common/StrategyItemContainer/StrategyItemContainer.tsx

```diff
@@ -112,33 +114,39 @@ export const StrategyItemContainer: FC<StrategyItemContainerProps> = ({
                             </DragIcon>
                         )}
                     />
-                    <StyledHeaderContainer>
-                        <StrategyHeaderLink>
-                            <StringTruncator
-                                maxWidth='400'
-                                maxLength={15}
-                                text={formatStrategyName(String(strategy.name))}
-                            />
-                            <ConditionallyRender
-                                condition={Boolean(strategy.title)}
-                                show={
-                                    <StyledCustomTitle>
+                    <StrategyHeaderLink>
+                        <StyledHeaderContainer>
+                            {strategy.title ? (
+                                <>
+                                    <p>
                                         {formatStrategyName(
-                                            String(strategy.title),
+                                            String(strategy.name),
                                         )}
-                                    </StyledCustomTitle>
+                                        :
+                                    </p>
+                                    <Typography
+                                        component={`h${strategyHeaderLevel}`}
+                                    >
+                                        {strategy.title}
```

Comment:
Not sure why we tried to format the strategy title as a strategy name before. pretty sure that's an oversight. If you call your strategy "flexibleRollout", you probably want to see that and not "Gradual rollout" 💁🏼

Reply:
Aren't they essentially the same thing, with "Gradual rollout" being the friendly, more readable, name?

Reply:
Oh I see what you mean now. Yes, looks like an oversight, users should be able too call their strategies whatever they want 😄